workflows/rnaseq/config/rnaseq_patterns.yaml

```diff
@@ -11,9 +11,9 @@ libsizes:
   bam:     'data/rnaseq_samples/{sample}/{sample}.cutadapt.bam.libsize'
 fastq_screen: 'data/rnaseq_samples/{sample}/{sample}.cutadapt.screen.txt'
 featurecounts:
-  s0: 'data/rnaseq_samples/{sample}/{sample}.cutadapt.bam.featurecounts.s0.txt'
-  s1: 'data/rnaseq_samples/{sample}/{sample}.cutadapt.bam.featurecounts.s1.txt'
-  s2: 'data/rnaseq_samples/{sample}/{sample}.cutadapt.bam.featurecounts.s2.txt'
+  s0: 'data/rnaseq_samples/combined/cutadapt.bam.featurecounts.s0.txt'
```

Comment:
There is an existing `rnaseq_aggregation` directory, created for e.g. the multiqc rule (see [this pattern](https://github.com/lcdb/lcdb-wf/blob/master/workflows/rnaseq/config/rnaseq_patterns.yaml#L24)). I think these new aggregated counts would be better living there.

Reply:
(fixed in e867fd7)